### ISSUE TITLE
A11y: some graphical controls without labels do not provide tooltips 

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#68362]
**Steps to reproduce**:
Manage dashboards and folders - browse dashboards

**Actual Result**:
Some graphical controls without labels do not provide tooltips - neither when hovering with the mouse nor when receiving the keyboard focus, see attached picture.

**Expected Result**
All non-text content that is presented to the user has a text alternative (e.g. tooltip) that serves the equivalent purpose.

**Relevant WCAG Criteria:** [1.1.1 WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
https://www.w3.org/TR/WCAG21/#x1-1-1-non-text-content 

**Environment**:
- Grafana version: 9.1.0
- Data source type & version:
- User OS & Browser: Windows 10, Chrome Version 107.0.5304.88
- Others:

![1 1 1_browse-dashboards](https://github.com/grafana/grafana/assets/122528177/c0e13570-80c1-4ced-b134-f66e394e9f74)

**Comment by usmangt** [comment]
Hi @ChristofSimon,

Thanks for opening this issue.

I will forward this to our Engineers so that they can have a look and might be able to tell more details about it.

**Comment by tskarhed** [comment]
This is a generic issue with the RadioButton component. When there is no label, and there is an aria-label, the aria-label should be displayed in a Tooltip.

**Comment by tskarhed** [comment]
Using `description` instead of `ariaLabel` is probably preferred. `description` is placed in the title attribute of the label element
![image](https://github.com/grafana/grafana/assets/1438972/871d92bd-3313-4433-ba5f-b6370eeed696)
